sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Release History
 
 ## 1.0.0-beta.5 (Unreleased)
+- Renamed `CommunicationTokenRefreshOptions.getRefreshProactively()` to `CommunicationTokenRefreshOptions.isRefreshProactively()`
```

Comment:
Do we need to update the date to 2021-03-09? I think it was mentioned in the standup

Reply:
Thank you Minnie! Changing it right now :)

Reply:
We normally do that as the last pr before release. The (Unreleased) tag will prevent the pipeline from releasing. Change it early is actually a risk. :-)